CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # NVIDIA Container Toolkit Changelog
 
+* Add support for injecting /dev/nvidia-nvswitch* devices if the NVIDIA_NVSWITCH=enabled envvar is specified.
```

Comment:
Is this required or was it simpler to backport if this was also added?

Reply:
It is not strictly required. I can omit this.

Upon a closer look, I also need to cherry-pick [this commit](https://github.com/NVIDIA/nvidia-container-toolkit/commit/833254fa5937e2c18c775811522ded6fe8ffa975 ), as `gated.go` depends on some of it's refactoring. So this cherry-pick is a bit more intrusive than I originally thought.

Reply:
I don't think that commit is required. The `image.Getenv` conditionals such as:
```
if image.Getenv(nvidiaGDSEnvvar) == "enabled" {
```
can be replaced by:
```
if gds, _ := image[nvidiaGDSEnvvar]; gds == "enabled" {
```

We would need to backport the driverRoot vs devRoot split -- which may be more problematic. Knowing the nature of the change, it may be simpler to *backport* the functionality and trigger it in a similar way to how we're currently handling the GDS and MOFED features on this branch.